awesome_owl/static/src/todo_list/todo.js

```diff
@@ -0,0 +1,13 @@
+export class Todo {
+    static nextId = 1;
+
+    constructor(description) {
+        this.id = Todo.nextId++;
+        this.description = description;
+        this.isCompleted = false;
+    }
+
+    toggleState() {
+        this.isCompleted = !this.isCompleted
+    }
```

Comment:
I think the tutorial asks you to do it via a callback like `remove` (even if it arguably doesn't make much sense here)

\+ nitpick:
```suggestion
    toggleState() {
        this.isCompleted = !this.isCompleted;
    }
```